lib/console/console.coffee

```diff
@@ -89,6 +89,9 @@ class Console
       p = p.splitDown()
       p.setFlexScale 1/2
     p.activateItem @view
+    p.onDidActivate =>
+      console.log "calling focus"
+      @view.focusInput(true)
```

Comment:
You could perhaps try wrapping this line in a `setTimeout`. You can test it with, say, a second, and then reduce the delay to something reasonable (0 might even work).